File: Formations/Boucles/Random Loops/Solutions/1/2.java
class Solution {
    public static void main(String[] args) {
        int s = 0;
        for(int i=1;i<=991;i+=2) s+=i;
        for(int i=3;i<=99;i+=2) s+=i;
        System.out.println("Resultat ="+s);
    }
}

File: Formations/Boucles/Random Loops/Solutions/1/2.pas
program solution;
uses wincrt;
var
    s,i:Integer;
Begin
    s:=0;
    i:=1;
    while (i<=991) do
    begin
       s:=s+i;
       i:=i+2;
    end;
    i:=3;
    while (i<=99) do
    begin
       s:=s+i;
       i:=i+2;
    end;
    WriteLn('Resultat=',S);
end.
